### ISSUE TITLE
Views exposed filters missing `for` attribute on `label`s

### ISSUE THREAD

**Issue by aslaymoore** [az-digital/az_quickstart#2221]
## Problem/Motivation
If you include an exposed filter in a View (a checkbox group of taxonomy terms for example), the filter `label` is missing the `for` attribute, which matches up the `label` with the `input`. This prevents the entire checkbox from being clickable (usability and [accessibility problem](https://www.w3.org/WAI/tutorials/forms/labels/#associating-labels-explicitly)).

This seems like it might be a [Barrio issue](https://git.drupalcode.org/project/bootstrap_barrio/-/blob/8.x-4.30/templates/form/form-element-label.html.twig), but should be addressed in QS for the sake of accessibility and usability.

### To Reproduce

1. Add an exposed filter of taxonomy terms to a View (such as Person). Check the box to "allow multiple".
2. In the Better Exposed Filter settings, change the widget for the exposed filter to "checkboxes".

## Proposed resolution
Include the `for` attribute for the input `label`.

### Expected behavior
- Entire input (including checkbox label) should be clickable.
- Label should include a `for` attribute for accessibility

**Comment by ewlyman** [comment]
This looks resolved, very likely by the introduction of the az_finder module. 
Do we know if this needs to remain open?
